tpm2/error_test.go

```diff
@@ -0,0 +1,79 @@
+package tpm2
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestError(t *testing.T) {
+	var err error
+
+	// Vendor
+	err = decodeResponse(0x501)
+	ve, ok := err.(VendorError)
+	if !ok {
```

Comment:
Use reflect.DeepEqual here and for all other comparisons

Reply:
Great idea. Went a little further with that to reduce duplication. Thanks